tests/run_tests.py

```diff
@@ -63,26 +63,30 @@ def _test_cpp_aot(test_filename, build_dir, test_info):
 
 def _test_cpp():
     curr_dir = os.path.dirname(os.path.abspath(__file__))
-    if platform.system() == "Windows":
-        cpp_test_filename = 'taichi_cpp_tests.exe'
-        capi_test_filename = 'taichi_c_api_tests.exe'
-        build_dir = os.path.join(curr_dir, '../bin')
+    cpp_test_filename = 'taichi_cpp_tests.exe'
```

Comment:
Did we accidentally skip c api tests on *nix machines by doing this? log from the linux gpu job ;) 
```
2022-09-12T09:28:20.5094707Z Not found /home/dev/tests/../build/taichi_c_api_tests.exe, skipping...
2022-09-12T09:28:20.5095195Z Not found /home/dev/tests/../build/taichi_cpp_tests.exe, skipping...
```